content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,329 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
```

Comment:
These are not legit tags. Tags must come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

Reply:
Hi @SSwiniarski thank you for your review. Are you referring to everything on lines 5-8? If so, what would you recommend or would you have any suggestions on what I should replace with?